src/StreamJsonRpc.Desktop/MultilingualResources/StreamJsonRpc.Desktop.cs.xlf

```diff
@@ -148,6 +148,10 @@
           <source>This operation is not allowed before listening for messages has started.</source>
           <target state="translated">Před zahájením naslouchání zprávám není tato operace povolená.</target>
         </trans-unit>
+        <trans-unit id="ParameterNotObject" translate="yes" xml:space="preserve">
+          <source>Parameter is not in the form of a single object</source>
+          <target state="new">Parameter is not in the form of a single object</target>
```

Comment:
I'm not sure what I need to do to localize the new resource I added.  It's in English everywhere.  :(